core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.ts

```diff
@@ -83,7 +83,7 @@ export class DotEditContentFormComponent implements OnInit {
             this.onFormChange(value);
         });
 
-        this.formData.fields.forEach((field) => {
+        this.formData.contentType.fields.forEach((field) => {
```

Comment:
Does this mean that we are passing fields twice? `formData.fields` and `formData.contentType.fields`?

Reply:
And why the change?

Reply:
Because a change in `EditContentPayload` before in that interface we had a property called  `layout` `fields` and  `contentType` this last one as a string refereing to the conten type variable. All this props are actually in one single object, `DotCMSContentType`, so move that to a single prop.